config/config.go

```diff
@@ -82,6 +90,14 @@ func (l *Local) Validate() []error {
 		errors = append(errors, err)
 	}
 
+	if 0 == len(errors) {
```

Comment:
This check is dependent on the order of validations. This shouldn't happen as this piece of validation might as well become first inline in the future.

Instead, change your condition to be independent from the `errors` array by validating that the issue tracker is not empty:
```
if l.IssueTracker != "" {
   ...
}
```

Reply:
fixed in f4977d0